_ontimize_web_ngx_map/configuration/04-o-geojson-layer.component.md

```diff
@@ -0,0 +1,182 @@
+---
+permalink: /map/configuration/o-geojson-layer/overview
+title: "GeoJson Layer"
+comp: ogeojsonlayer
+layout: o-component
+
+---
+
+
+## Introduction
+
+
+You can add multiple WMS layers to your map.
+
+
+## Basic Example
+
+This is a basic example of a *Map* using the component *o-map-layer* inside to add new WMS layers to the map.
```

Comment:
This is the GeoJson layer documentation file but the description is about WMS layers.